### DISCUSSION TITLE
Bug on Github Contributors Page

### DISCUSSION THREAD

**Discussion by parikshittestbook** [community/community#87698] [Repositories]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

`/${repo-slug}/graphs/contributors`
Showing random months 
<img width="540" alt="Screenshot 2024-01-15 at 11 50 44 PM" src="https://github.com/community/community/assets/126481026/497ab555-7888-4c58-8535-f51add2b161f">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @parikshittestbook ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
